peachjam/resources.py

```diff
@@ -265,6 +265,22 @@ def download_attachment(url, document, nature):
     def before_import_row(self, row, **kwargs):
         logger.info(f"Importing row: {row}")
 
+        frbr_uri_data = {
```

Comment:
It's annoying that we have to do this before the row has been validated and cleaned :/

Bear in mind we can only generate an expression FRBR URI if the serial number override is provided; otherwise, skip this whole thing

Reply:
@longhotsummer Yes. We need the serial number override for this to work.  It does ruin some of the validations that we had setup. I am testing to see the behavior now For the generic documents, we had the work_frbr_uri in the spreadsheet. Should I assume that this is still the case or should we try to generate it in the importer? Also are Mariya and the team still using the importer for generic documents?